src/sentry/replays/usecases/ingest/dead_click.py

```diff
@@ -57,6 +57,6 @@ def _report_dead_click_issue(
         project_id=project_id,
         subtitle=subtitle,
         timestamp=timestamp,
-        title="[TEST] Dead Click Detected",
+        title="Possible Dead Click Detected",
```

Comment:
I feel "possible" and "detected" right after feels a bit off, how about:

```suggestion
        title="Suspected Dead Click",
```

Reply:
Detected Possible Suspect Dead Click Maybe